ios_app_clip/.metadata

```diff
@@ -0,0 +1,10 @@
+# This file tracks properties of this Flutter project.
+# Used by Flutter tool to assess capabilities and perform upgrades etc.
+#
+# This file should be version controlled and should not be manually edited.
+
+version:
+  revision: 65ae30c8cc99b7d48b5bd08d3726f5669b819be0
+  channel: master
```

Comment:
If this is targeting master (and I'm guessing it should), it should go in `/experimental` rather than the root directory.

Reply:
Oh good point. This will work fine on 1.20 actually. I just happened to have made this on master.